lte/gateway/c/core/oai/test/s6a_task/BUILD.bazel

```diff
@@ -0,0 +1,24 @@
+# Copyright 2022 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+load("@rules_cc//cc:defs.bzl", "cc_test")
+
+package(default_visibility = ["//visibility:private"])
+
+cc_test(
+    name = "s6a_recv_async_grpc_messages_test",
```

Comment:
Can we set size="small" here? (Scott and I were just saying how it would be nice to have a linter to prompt devs to add test size for timeouts)

Reply:
Sounds good :)

Reply:
I force-pushed, but Github doesn't seem to pick up my new commit. Maybe it needs a bit more time :shrug:

Reply:
@jheidbrink , can you try force push once more? Then I can approve and merge this one.

Reply:
It seems Github eventually picked up the push :)